src/main/java/com/google/opensesame/projects/ProjectData.java

```diff
@@ -10,57 +10,105 @@
 import java.util.List;
 import java.util.Map;
 import org.kohsuke.github.GHRepository;
-import org.kohsuke.github.GitHub;
 
 /**
- * A class containing all relevant data for a project. This is intended to be serialized and sent to
- * the client.
+ * A class containing the data for a project. This is intended to be serialized and sent to the
+ * client. A ProjectData object can define as many or as few of the fields as necessary depending on
+ * the use case, as undefined values will not be serialized to JSON and will therefore not be sent
+ * to the client.
```

Comment:
Added a comment below.

This information is useful, but not sure if it's the first thing to write in class documentation. It's kind of an implementation detail. As we've discussed, it's important to clearly define the use-cases i.e. "why" would I use this class/what would I use it for?

Reply:
A stab at an example docstring: "Read-only utility class for converting a ProjectEntity datastore object into a POJO for JSON serialization" 

:) something like that haha sorry for throwing in the POJO jargon.

Reply:
I've seen POJO being thrown around before in Objectify, so no worries there! I'll update this to reflect more of the "why" when I push my other changes.